src/common/backend/services/wallabag/form.tsx

```diff
@@ -0,0 +1,104 @@
+import { Form } from '@ant-design/compatible';
+import '@ant-design/compatible/assets/index.less';
+import { Input } from 'antd';
+import { FormComponentProps } from '@ant-design/compatible/es/form';
+import React, { Fragment } from 'react';
+import useOriginForm from '@/hooks/useOriginForm';
+import { FormattedMessage } from 'react-intl';
+import { WallabagBackendServiceConfig } from 'common/backend/clients/wallabag/interface';
+
+interface WallabagFormProps {
+  verified?: boolean;
+  info?: WallabagBackendServiceConfig;
+}
+
+const FormItem: React.FC<WallabagFormProps & FormComponentProps> = props => {
+  const {
+    form,
+    form: { getFieldDecorator },
+    info,
+    verified,
+  } = props;
+
+  const { verified: formVerified, handleAuthentication, formRules } = useOriginForm({
+    form,
+    initStatus: !!info,
+  });
+
+  let initData: Partial<WallabagBackendServiceConfig> = {};
+  if (info) {
+    initData = info;
+  }
+  let editMode = info ? true : false;
+  return (
+    <Fragment>
+      <Form.Item
+        label={
+          <FormattedMessage id="backend.services.confluence.form.origin" defaultMessage="Origin" />
+        }
+      >
+        {form.getFieldDecorator('wallabag_host', {
```

Comment:
field name must be `origin`

Reply:
pr set `originKey `
https://github.com/webclipper/web-clipper/blob/eebc4a2ecf34fda83d5c32620fd68b1c52412f70/src/hooks/useOriginForm.tsx#L12